package.json

```diff
@@ -45,9 +45,17 @@
     "api-documenter": "api-documenter yaml --input-folder=temp"
   },
   "dependencies": {
+    "fast-crc32c": "^2.0.0",
```

Comment:
this dependency should be added to `samples/package.json`, vs., `package.json`, at which point if you run:

```
rm -rf node_modules
rm package-lock.json
rm -rf samples/package-lock.json
rm -rf samples/node_modules
npm run i
npm run lint
```

From the root directory of the library, I believe you'll stop seeing the `no-extraneous-require` linting issues.

Reply:
Done.